Statistics.Test/StatsUnitTest.cs

```diff
@@ -10,18 +11,20 @@ public class StatsUnitTest
         public void ReportsAverageMinMax()
         {
             var statsComputer = new StatsComputer();
-            var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{1.5, 8.9, 3.2, 4.5});
-            float epsilon = 0.001F;
-            Assert.True(Math.Abs(statsComputer.average - 4.525) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.max - 8.9) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.min - 1.5) <= epsilon);
+            var computedStats = statsComputer.CalculateStatistics( new List<double> {1.5, 8.9, 3.2, 4.5} );
```

Comment:
You already return computedStats, so you can change the test to rely on that itself. Then you can remove the data members inside StatsComputer and keep it as a pure computer of stats only... not a storage for 'last known stats' as well.